api/src/exam-environment/routes/exam-environment.ts

```diff
@@ -565,3 +572,28 @@ async function postScreenshotHandler(
 ) {
   return reply.code(418);
 }
+
+async function getExams(
+  this: FastifyInstance,
+  req: UpdateReqType<typeof schemas.examEnvironmentExams>,
+  reply: FastifyReply
+) {
+  const user = req.user!;
+  const exams = await this.prisma.envExam.findMany();
+
+  const availableExams = exams.map(exam => {
+    const isExamPrerequisitesMet = checkPrerequisites(user, true);
+
+    return {
+      examId: exam.id,
```

Comment:
This is explicit enough, seeing as we are returning `exams[].id`

```suggestion
      id: exam.id,
```